RELEASE_NOTES.md

```diff
@@ -1,5 +1,43 @@
 # Release Notes
 
+## Changes in Splinter 0.5.14
+
+### Highlights
+
+* Splinterd will not start if there are database migrations that need to be run
+  and will give information about how to upgrade the database.
+
+* User configurable logging with stdout, stderr, file, and rotating file log
+  targets as well as fine grain control over where each log statement goes.
+
+### libsplinter
+
+* Replace the use of a Vec<u8> to represent a public key in the
+  PeerAuthorizationToken, ConnectionAuthorizationType, Identity, and in the
+  AdminService.
+
+* Add any_pending_migrations to postgres and sqlite migration modules
+
+* Fix identity comparison in PeerManager to decide between inbound/outboud
+  connections. This comparison must now be between the connection's identity
+  and the required local authorization used when requesting the connection
+  originally.
+
+* Stabilize `admin-service-count`. This makes `count_circuits` and
+  `count_proposals`available on the     `AdminServiceStore`.
+
+### splinterd
+
+* Detect pending migrations and abort if migrations are required
+
+* Add `--peering-key` option to the splinterd to set which key will be used
```

Comment:
The peering ones are under an experimental feature, which should be noted in both entries

Reply:
Added some language to that effect on both entries